lectures/lecture01.ipynb

```diff
@@ -28,6 +28,16 @@
     "### Comparison of execution speed of Julia, Java, Python, and C"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08d63bd7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Pkg.activate(Base.current_project())"
```

Comment:
why is this needed? Also - does it work (i.e. without using `using Pkg` first?

Reply:
> does it work (i.e. without using `using Pkg` first?

With the Dev Containers: Yes. Because of https://github.com/b-data/julia-docker-stack/blob/e0b072105d6b12c0722b5f104320fee7daba1f22/base/conf/julia/etc/julia/startup.jl#L8.

But not in general. I will add `using Pkg`.

> why is this needed?

To make absolutely sure, it works across different setups and configurations.

In Jupyter Notebooks, project environments are not automatically activated in every case.

Reply:
>  I will add using Pkg.

Thanks

> In Jupyter Notebooks, project environments are not automatically activated in every case.

Indeed this is the case.